lib/arithmetic.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
Read Feature Tasks carefully. There should be something that happens if a user passes non-numbers as arguments.